product_docs/docs/bdr/4/appusage.mdx

```diff
@@ -1,130 +1,124 @@
 ---
-title: Application Usage
+title: Application use
 ---
 
-This chapter looks at BDR from an application or user perspective.
+Learn about the application from a user perspective.
 
-Setting up nodes is discussed in a later chapter, as is replication
-of DDL, and various options for controlling replication using
-replication sets.
-
-## Application Behavior
+## Application behavior
 
 BDR supports replicating changes made on one node to other nodes.
 
-BDR will, by default, replicate all changes from INSERTs, UPDATEs, DELETEs
-and TRUNCATEs from the source node to other nodes. Only the final changes
-will be sent, after all triggers and rules have been processed. For example,
-INSERT ... ON CONFLICT UPDATE will send either an INSERT or an UPDATE
-depending on what occurred on the origin. If an UPDATE or DELETE affects
-zero rows, then no changes will be sent.
+BDRs, by default, replicate all changes from INSERT, UPDATE, DELETE
```

Comment:
There is no BDRs, BDR should always be used as singular. One can say "BDR nodes", but not BDRs.